easybuild/easyconfigs/s/Sphinx/Sphinx-1.8.3-foss-2019a-Python-3.7.2.eb

```diff
@@ -0,0 +1,78 @@
+easyblock = 'PythonBundle'
+
+name = 'Sphinx'
+version = '1.8.3'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://sphinx.pocoo.org/'
+description = """Sphinx is a tool that makes it easy to create intelligent and beautiful documentation.
+ It was originally created for the new Python documentation, and it has excellent facilities for the 
+ documentation of Python projects, but C/C++ is already supported as well, and it is planned to add 
+ special support for other languages as well."""
+
+toolchain = {'name': 'foss', 'version': '2019a'}
+
+dependencies = [('Python', '3.7.2')]
+
+use_pip = True
+
+exts_default_options = {'source_urls': [PYPI_SOURCE]}
+exts_list = [
+    ('packaging', '19.0', {
+        'checksums': ['0c98a5d0be38ed775798ece1b9727178c4469d9c3b4ada66e8e6b7849f8732af'],
+    }),
+    ('MarkupSafe', '1.1.0', {
+        'checksums': ['4e97332c9ce444b0c2c38dd22ddc61c743eb208d916e4265a2a3b575bdccb1d3'],
+    }),
+    ('sphinxcontrib-websupport', '1.1.0', {
+        'modulename': 'sphinxcontrib.websupport',
+        'use_pip': False,
+        'checksums': ['9de47f375baf1ea07cdb3436ff39d7a9c76042c10a769c52353ec46e4e8fc3b9'],
+    }),
+    ('Pygments', '2.3.1', {
+        'checksums': ['5ffada19f6203563680669ee7f53b64dabbeb100eb51b61996085e99c03b284a'],
+    }),
+    ('imagesize', '1.1.0', {
+        'checksums': ['f3832918bc3c66617f92e35f5d70729187676313caa60c187eb0f28b8fe5e3b5'],
+    }),
+    ('Docutils', '0.14', {
+        'source_tmpl': 'docutils-%(version)s.tar.gz',
+        'source_urls': [('http://sourceforge.net/projects/docutils/files/docutils/%(version)s/', 'download')],
+        'checksums': ['51e64ef2ebfb29cae1faa133b3710143496eca21c530f3f71424d77687764274'],
+    }),
+    ('Jinja2', '2.10', {
+        'checksums': ['f84be1bb0040caca4cea721fcbbbbd61f9be9464ca236387158b0feea01914a4'],
+    }),
+    ('snowballstemmer', '1.2.1', {
+        'checksums': ['919f26a68b2c17a7634da993d91339e288964f93c274f1343e3bbbe2096e1128'],
+    }),
+    ('Babel', '2.6.0', {
+        'checksums': ['8cba50f48c529ca3fa18cf81fa9403be176d374ac4d60738b839122dfaaa3d23'],
+    }),
+    ('alabaster', '0.7.12', {
+        'checksums': ['a661d72d58e6ea8a57f7a86e37d86716863ee5e92788398526d58b26a4e4dc02'],
+    }),
+    (name, version, {
+        'checksums': ['c4cb17ba44acffae3d3209646b6baec1e215cad3065e852c68cc569d4df1b9f8'],
+    }),
+    ('sphinx_rtd_theme', '0.4.2', {
+        'checksums': ['02f02a676d6baabb758a20c7a479d58648e0f64f13e07d1b388e9bb2afe86a09'],
+    }),
+    ('numpydoc', '0.8.0', {
+        'checksums': ['61f4bf030937b60daa3262e421775838c945dcdd671f37b69e8e4854c7eb5ffd'],
+    }),
+    ('sphinx-bootstrap-theme', '0.6.5', {
+        'checksums': ['82936109a05b84029052e7efb1756161c3b4c27ae18512a90234d50fe608cba7'],
+    }),
+    ('sphinx-gallery', '0.2.0', {
+        'checksums': ['b49356b5516cc7dab67b1b378f6bf8146fe2372ee73d5e1ea2c483a2e3f4f182'],
+    }),
+]
+
+sanity_pip_check = False
```

Comment:
@zarybnicky The error message in the tests is a bit confusing, but this should be `sanity_pip_check = True`.

We want to run `pip check` in the sanity check step to ensure that all required dependencies for Sphinx are installed.